xcube_cci/constants.py

```diff
@@ -38,3 +38,6 @@
 DEFAULT_NUM_RETRIES = 200
 
 CCI_MAX_IMAGE_SIZE = 2500
+
+STANDARD_COORD_VAR_NAMES = ['time', 'lat', 'lon', 'latitude', 'longitude',
```

Comment:
Rename to `COMMON_COORD_VAR_NAMES`? Because these are just common but not standard names.

Reply:
Okay!